lib/tasks/translate_collections.rake

```diff
@@ -0,0 +1,277 @@
+require 'libxml_to_hash'
+
+namespace :collection do
+  desc 'Translate a collection from native format to UMM JSON and back to native format'
+  task :translate, [:file, :format, :disp, :version] => :environment do |_task, args|
+    args.with_defaults(:version => '1.15.3')
+    args.with_defaults(:disp => 'show')
+
+    puts 'FORMAT INVALID' unless args.format == ('echo10' || 'dif10' || 'iso19115')
```

Comment:
Should fail gracefully here if the format is invalid o/w results text can push the error out of the window and the user may not know that it was invalid.